unsloth/models/llama.py

```diff
@@ -763,6 +766,12 @@ def LlamaModel_fast_forward(
         pass
     pass
 
+    
+    if IS_GRANITE:
```

Comment:
So this is a must must?

Reply:
Yeah iirc, Granit's forward [calculates it here and passes on](https://github.com/huggingface/transformers/blob/4d1d0f29a493098e6bc6b904b82e29cb331827f5/src/transformers/models/granite/modeling_granite.py#L812) and without this it throws error (I don't exactly remember the error unfortunately)